### PR TITLE
Convert qenum key tests

### PR DESCRIPTION
**Pull request by hackebrot**
Convert TestCase `QEnumKeyTests` to `pytest`. #10

### CODE REVIEW

**Comment by nicoddemus** [comment]
Nice work! :smile:

@The-Compiler feel free to merge if you are OK with this change, but mind that your input in `test_reconverted` is welcome.

File: qutebrowser/test/utils/debug/test_qenum_key.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    """Test passing an unknown value."""
+
+    key = debug.qenum_key(QFrame, 0x1337, klass=QFrame.Shadow)
+    assert key == '0x1337'
+
+
+def test_reconverted():
+    """Test passing a flag value which was re-converted to an enum."""
+
+    pytest.skip(msg="It is not clear what this test is supposed to do")
```
Comment (The-Compiler) [inline qutebrowser/test/utils/debug/test_qenum_key.py]: That... is a damn good question :blush:. The only thing I can imagine is this causing an exception in earlier versions, and here I wanted to check there's no exception. I'll test with an older version of `qenum_key` to see what happens :)

File: qutebrowser/test/utils/debug/test_qenum_key.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+from qutebrowser.utils import debug
+
+
+def test_no_metaobj():
+    """Test with an enum with no meta-object."""
+
+    with pytest.raises(AttributeError):
+        # Make sure it doesn't have a meta object
+        # pylint: disable=pointless-statement,no-member
+        QStyle.PrimitiveElement.staticMetaObject
```
Comment (The-Compiler) [inline qutebrowser/test/utils/debug/test_qenum_key.py]: I just noticed a `assert not hasattr(QStyle.PrimitiveElement, 'staticMetaObject')` would probably be much clearer here (and similiar in `test_metaobj()`).

**Comment by The-Compiler** [comment]
I removed `test_reconverted`, switched the attribute tests to `hasattr` as noted in my comment, did some small style adjustments and merged this! :+1:

**Pull request merged**